py-polars/tests/unit/test_interop.py

```diff
@@ -604,7 +604,7 @@ def test_from_empty_arrow() -> None:
     assert df.dtypes == [pl.Float64, pl.Float64]
 
     # 2705
-    df1 = pd.DataFrame(columns=["b"], dtype=float)
+    df1 = pd.DataFrame(columns=["b"], dtype=float, index=pd.Index([]))
     tbl = pa.Table.from_pandas(df1)
```

Comment:
empty dataframes now have a RangeIndex by default: https://pandas.pydata.org/docs/dev/whatsnew/v2.0.0.html#empty-dataframes-series-will-now-default-to-have-a-rangeindex

in pyarrow https://arrow.apache.org/docs/python/generated/pyarrow.Table.html :

> The default of None will store the index as a column, except for RangeIndex which is stored as metadata only. Use preserve_index=True to force it to be stored as a column.